diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/efax-0.9a-001114/Makefile src/efax-0.9a-001114/Makefile
--- origsrc/efax-0.9a-001114/Makefile	2000-06-28 21:01:22.000000000 -0500
+++ src/efax-0.9a-001114/Makefile	2007-06-26 16:25:06.104135700 -0500
@@ -25,16 +25,13 @@
 
 efax:	efax.o efaxlib.o efaxio.o efaxos.o efaxmsg.o
 	$(CC) -o efax $(LDFLAGS) efax.o efaxlib.o efaxio.o efaxos.o efaxmsg.o
-	strip efax
 
 efix:	efix.o efaxlib.o efaxmsg.o
 	$(CC) -o efix $(LDFLAGS) efix.o efaxlib.o efaxmsg.o
-	strip efix
 
 install:
-	cp fax efax efix $(BINDIR)
-	chmod 755 $(BINDIR)/fax $(BINDIR)/efax $(BINDIR)/efix
-	cp fax.1 efax.1 efix.1 $(MANDIR)/man1
+	install -m755 fax efax efix $(BINDIR)
+	install -m644 fax.1 efax.1 efix.1 $(MANDIR)/man1
 	chmod 644 $(MANDIR)/man1/fax.1 $(MANDIR)/man1/efax.1  \
 		$(MANDIR)/man1/efix.1
 
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/efax-0.9a-001114/efaxlib.h src/efax-0.9a-001114/efaxlib.h
--- origsrc/efax-0.9a-001114/efaxlib.h	2000-02-06 13:56:26.000000000 -0600
+++ src/efax-0.9a-001114/efaxlib.h	2007-06-26 16:24:09.959136600 -0500
@@ -91,7 +91,7 @@
 		"DFAX", "PCX", "RAW", "DCX" } ;
 
 enum oformats { O_AUTO=0, O_PBM=1, O_FAX=2, O_PCL=3, O_PS=4, 
-		O_PGM=5, O_TEXT=6, O_TIFF_FAX=7, O_TIFF_RAW=8, O_DFAX=9, 
+		O_PGM=5, O_TXT=6, O_TIFF_FAX=7, O_TIFF_RAW=8, O_DFAX=9, 
 		O_TIFF=10, O_PCX=11, O_PCX_RAW=12, O_DCX=13 } ;
 
 #define OFORMATS { "AUTO", "PBM", "FAX", "PCL", "PS", \
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/efax-0.9a-001114/efaxos.h src/efax-0.9a-001114/efaxos.h
--- origsrc/efax-0.9a-001114/efaxos.h	1999-02-18 21:15:30.000000000 -0600
+++ src/efax-0.9a-001114/efaxos.h	2007-06-26 16:23:25.824929400 -0500
@@ -8,7 +8,7 @@
 /* signals to be caught */
 
 #define ANSISIGS  SIGABRT, SIGFPE, SIGILL, SIGINT, SIGSEGV, SIGTERM
-#define UNIXSIGS  SIGHUP, SIGQUIT, SIGIOT, SIGALRM
+#define UNIXSIGS  SIGHUP, SIGQUIT, SIGALRM
 #define CATCHSIGS ANSISIGS, UNIXSIGS
 
 /* Bit order reversal table. */
